docs/content/operations/performance-faq.md

```diff
@@ -20,6 +20,21 @@ We recommend 250mb * (processing.numThreads) for the heap.
 
 [Coordinator nodes](../design/coordinator.html) do not require off-heap memory and the heap is used for loading information about all segments to determine what segments need to be loaded, dropped, moved, or replicated.
 
+## How much direct memory does Druid use?
+Indexing workers and historical nodes use two kinds of direct memory buffers with configurable size: processing buffers and merge buffers.
```

Comment:
Replace "Indexing workers and historical nodes" with "Any Druid node that handles queries" (broker is included too)